src/Compilers/CSharp/Portable/CodeGen/CodeGenerator.cs

```diff
@@ -311,6 +311,11 @@ private void EmitSymbolToken(TypeSymbol symbol, CSharpSyntaxNode syntaxNode)
             _builder.EmitToken(_module.Translate(symbol, syntaxNode, _diagnostics), syntaxNode, _diagnostics);
         }
 
+        private void EmitImplementationSymbolToken(Cci.ITypeReference symbol, CSharpSyntaxNode syntaxNode)
```

Comment:
Alternatively, this could be called something like "EmitTypeReferenceToken" and `EmitSymbolToken` could delegate to it.